IDEAS/Examples/Tutorial/DetailedHouse/package.mo

```diff
@@ -0,0 +1,107 @@
+within IDEAS.Examples.Tutorial;
+package DetailedHouse "Package with example for how to built a simple house, using the detailed building envelope component
+  models, HVAC and other components"
+  extends Modelica.Icons.ExamplesPackage;
+
+  annotation (Documentation(info="<html>
+<p>
+This package contains examples with step-by-step instructions for how to build a system model
```

Comment:
The step-by-step instructions are currently not fully included in the documentation of each model, thus I would either add this or clearly refer to the instructions in the crash course repo (https://github.com/open-ideas/__CrashCourse__/blob/master/Exercises/Exercise%203/Latex/Exercise3.pdf and https://github.com/open-ideas/__CrashCourse__/blob/master/Exercises/Exercise%203/Latex/Exercise4.pdf).